tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1724,8 +1724,23 @@ def convert_image_dtype(image, dtype, saturate=False, name=None):
 
   Returns:
     `image`, converted to `dtype`.
+  
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
+    >> x = tf.random.normal(shape=(256, 256, 3), dtype=tf.float32)
+    >> tf.image.convert_image_dtype(x, dtype=tf.float16, saturate=False)
+    ```
+    
+  Raises:
+    AttributeError: Raises an attribute error when dtype is neither float nor integer
   """
   image = ops.convert_to_tensor(image, name='image')
+  dtype = tf.dtypes.as_dtype(dtype)
+  if dtype.is_floating or dtype.is_integer:
```

Comment:
Can you rewrite this as

if not dtype.is_floating and not dtype.is_integer: raise ...?

Much easier to read without if...pass...else

Reply:
Hey. I made the changes.